__init__.py

```diff
@@ -60,6 +77,38 @@ def decompile_function(self):
         code = self.merge_symbols(code)
         self.render_output(code)
 
+    def read_file(self):
+        start = self._function.start
+
+        #Doesn't just operate on the original file since in a .BNDB the original file may no longer exist
+        fd, filename = tempfile.mkstemp()
+        with os.fdopen(fd, 'wb') as f:
+            f.write(self._view.file.raw.read(0, len(self._view.file.raw)))
+
+        self.cmdline.append('--select-ranges')
+        self.cmdline.append('{:#x}-{:#x}'.format(start, start+1))
```

Comment:
Don't know if this is the right way to pass the range. But I haven't tested it and it just seems odd.